### DISCUSSION TITLE
Unable to start aio if the connection to the default alpine mirror is too slow

### DISCUSSION THREAD

**Discussion by mio-19** [nextcloud/all-in-one#3989] [Questions] [answered]
### Steps to reproduce
1.Install and start AIO in an unfriendly network environment, China mainland for example

### Actual behavior 

The nextcloud-aio-nextcloud container is stuck:

```
Installing imagemagick via apk...
```

### Host OS 
Not relevant: Archlinux or Windows with Docker Desktop

#### Nextcloud AIO version 
7.9.0
#### Current channel 
latest
#### Other valuable info 
There should be an option to change alpine mirror or include imagemagick  to the container image

**Comment by szaimen** [comment, marked as answer]
In my opinion Alpine should fix their default mirror to be fast everywhere in the world. We also don't have any plans to make the mirror configurable. However you might be able to make it work with https://github.com/nextcloud/all-in-one/tree/main/manual-install
